### DISCUSSION TITLE
How to do only security patch update using rauc

### DISCUSSION THREAD

**Discussion by kunaljee12** [rauc/meta-rauc-community#138] [General]

**Comment by ejoerns** [comment]
This is more a [RAUC](https://github.com/rauc/rauc)-related question than for meta-rauc-community. You can patch your component, build a new disk image and deploy it with RAUC.

If you are looking for delta-like update strategies, have a look at https://rauc.readthedocs.io/en/latest/advanced.html#adaptive-updates
